test/E2E_Test/E2ETestBase.cs

```diff
@@ -12,24 +13,26 @@ namespace Microsoft.VisualStudio.Web.CodeGeneration.E2E_Test
 {
     public class E2ETestBase
     {
-        public static string BaseLineFilesDirectory = Path.Combine(Directory.GetCurrentDirectory(), "Baseline");
-
+        private static object syncObject = new object();
+        public string BaseLineFilesDirectory;
         public const string E2ESkipReason = "Disabling E2E test";
         public  const string codegeneratorToolName = "aspnet-codegenerator";
 
+        //protected string BaseLineFilesDirectory { get; }
         protected string TestProjectPath { get; set; }
         protected ITestOutputHelper Output { get; set; }
-        
 
         public E2ETestBase(ITestOutputHelper output)
         {
+            BaseLineFilesDirectory = Path.Combine(Directory.GetCurrentDirectory(), "Baseline");
```

Comment:
Why'd this have to change? You modify the working directory before calling into this now?